src/bus_monitor.cpp

```diff
@@ -10,6 +10,77 @@
 namespace panel
 {
 
+static void resetLEDState()
+{
+    std::cout << "Reset LED state" << std::endl;
+    std::vector<std::string> ledsPathOnBasePanel{
```

Comment:
Please check if this can be constexpr. We don't want to create the vector every time in the function

Reply:
This function will be called only when Everest base panel will se set from false to true i.e. only at panel CM.
How frequently this will be called? Just trying to understand if it will have much effect?